src/mpl/include/mpl_thread.h

```diff
@@ -12,11 +12,13 @@
 /* _INVALID exists to avoid accidental macro evaluations to 0 */
 #define MPL_THREAD_PACKAGE_INVALID 0
 #define MPL_THREAD_PACKAGE_NONE    1
-#define MPL_THREAD_PACKAGE_POSIX   2
-#define MPL_THREAD_PACKAGE_SOLARIS 3
-#define MPL_THREAD_PACKAGE_WIN     4
-#define MPL_THREAD_PACKAGE_UTI     5
-#define MPL_THREAD_PACKAGE_ARGOBOTS 6
+#define MPL_THREAD_PACKAGE_YES     2    /* marker: thread is supported */
+#define MPL_THREAD_PACKAGE_POSIX   3
+#define MPL_THREAD_PACKAGE_SOLARIS 4
+#define MPL_THREAD_PACKAGE_WIN     5
+#define MPL_THREAD_PACKAGE_NON_NATIVE 6 /* marker: no native compiler tls support */
+#define MPL_THREAD_PACKAGE_UTI      7
+#define MPL_THREAD_PACKAGE_ARGOBOTS 8
```

Comment:
It is extremely hard to understand (e.g., the meaning of `MPL_THREAD_PACKAGE_NAME < MPL_THREAD_PACKAGE_YES`).

Reply:
Do you like the previous alternative better? Which is to explicitly compare individual enum values. 

I find it equally obscure and also not scalable. I.e. every time we add or change supported thread-package, we need re-evaluate every such condition or we risk missing things.

Reply:
It is solved.